php/php.editor/src/org/netbeans/modules/php/editor/codegen/CGSInfo.java

```diff
@@ -346,6 +354,24 @@ public void visit(FieldsDeclaration node) {
             }
         }
 
+        private String getPropertyType(FieldsDeclaration fieldsDeclaration, SingleFieldDeclaration singleFieldDeclaration) {
+            String type = ""; // NOI18N
+            if (fieldsDeclaration.getFieldType() == null || phpVersion.compareTo(PhpVersion.PHP_74) < 0) {
+                type = getPropertyType(singleFieldDeclaration);
+            } else {
+                // PHP 7.4 or newer
+                QualifiedName qualifiedName = QualifiedName.create(fieldsDeclaration.getFieldType());
+                if (qualifiedName != null) {
+                    type = qualifiedName.toString();
+                    if (fieldsDeclaration.getFieldType() instanceof NullableType) {
+                        type = CodeUtils.NULLABLE_TYPE_PREFIX + type;
+                    }
+                }
+                assert !type.isEmpty();
```

Comment:
If this fails, some more information could be useful in order to fix the error (e.g. `qualifiedName` or `fieldsDeclaration` could be added?).